solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudHttp2SolrClientTest.java

```diff
@@ -100,7 +100,13 @@ public class CloudHttp2SolrClientTest extends SolrCloudTestCase {
   private static final Pattern PATTERN_WITHOUT_COLLECTION =
       Pattern.compile(
           "path=/admin/collections.*params=\\{[^}]*action=CLUSTERSTATUS"
-              + "(?![^}]*collection=)[^}]*\\}");
+              + "(?=[^}]*includeAll=true)"
```

Comment:
Regex pattern (sad) updated to reflect the new params to CLUSTERSTATUS call

Reply:
let's do away with this regex; it's gotten out of control and should cause us to re-think what we're doing (see my other comment)

Reply:
Updated test doesnt include regexes anymore.